### PR TITLE
feat(CI): update clang-7 to clang-9, add clang-10 build

### PR DESCRIPTION
**Pull request by FrancescoBorzi**
After this PR, our CI will do the following checks

- clang-6 that also checks warnings **(no changes)**
- clang-9 that also checks warnings **(updated clang-7 to clang-9)**
- clang-10 **without** warnings check **(new)**

We can enable clang-10 warnings check (and remove clang-9) once all warnings of clang-10 are solved (see https://github.com/azerothcore/azerothcore-wotlk/issues/3108).

### CODE REVIEW

**Comment by Kitzunu** [comment]
E: Unable to locate package clang-10

File: apps/ci/ci-install.sh [hunk 1]
```diff
@@ -30,10 +30,18 @@ case $COMPILER in
...
-    time sudo apt-get install -y clang-7
-    echo "CCOMPILERC=\"clang-7\"" >> ./conf/config.sh
-    echo "CCOMPILERCXX=\"clang++-7\"" >> ./conf/config.sh
+   "clang9" )
+    time sudo apt-get install -y clang-9
+    echo "CCOMPILERC=\"clang-9\"" >> ./conf/config.sh
+    echo "CCOMPILERCXX=\"clang++-9\"" >> ./conf/config.sh
+    ;;
+
+  "clang10" )
```
Comment (Kitzunu) [inline apps/ci/ci-install.sh:39]: missing one space

Comment (FrancescoBorzi) [reply apps/ci/ci-install.sh:39]: fixed, thanks

**Comment by Yehonal** [comment]
Good job

**Pull request merged**